### PR TITLE
Tao 2639 deliveries update script

### PR DESCRIPTION
**Pull request by hutnikau**

### CODE REVIEW

File: scripts/RecompileDelivery.php [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+\common_ext_ExtensionsManager::singleton()->getExtensionById('taoDeliveryRdf');
+
+/**
+ * Class RecompileDelivery
+ * @package oat\taoDeliveryRdf\scripts
+ * @author Aleh Hutnikau, <hutnikau@1pt.com>
+ *
+ * Run example:
+ * ```
+ * sudo -u www-data php index.php 'oat\taoDeliveryRdf\scripts\RecompileDelivery'
```
Comment (ssipasseuth) [inline scripts/RecompileDelivery.php]: maybe some real life example would help here

Comment (ssipasseuth) [inline scripts/RecompileDelivery.php]: Indeed am unsure about how I should use the script
```
sudo -u www-data php index.php 'oat\taoDeliveryRdf\scripts\RecompileDelivery' compile http://tao.local/mytao.rdf#i14646778081995158
PHP Fatal error:  Call to a member function getUri() on null in /home/sam/Workspace/projects/TAO/PR/taoDeliveryRdf/scripts/RecompileDelivery.php on line 147
```
where http://tao.local/mytao.rdf#i14646778081995158 is a delivery I just created and got the uri from
```
sudo -u www-data php index.php 'oat\taoDeliveryRdf\scripts\RecompileDelivery' list
http://tao.local/mytao.rdf#i14649684368987171 - Delivery of Test 3
http://tao.local/mytao.rdf#i14646088199931108 - Delivery of QTI Example Test
http://tao.local/mytao.rdf#i14646754904861141 - Delivery of Test 2
http://tao.local/mytao.rdf#i14646778081995158 - Delivery of Test 3

**Comment by hutnikau** [comment]
@ssipasseuth I've updated the branch. Please review it again.

**Comment by ssipasseuth** [comment]
Thanks, it works fine.
I am just wondering, woudn't it be also useful to have another script to recompile by a class recursively ?
I know that is not part of the task, but maybe something we can suggest as an additonal test.

**Pull request merged**